### PR TITLE
crash minion

### PR DESCRIPTION
**Pull request by zapion**
will add test for this

### CODE REVIEW

**Comment by zapion** [comment]
@ShakoHo , @askeing r?

**Comment by ShakoHo** [comment]
Just a reminder that we need to update the README.md

File: moz_minions/lib/b2g_crash_util.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/python
```
Comment (ShakoHo) [inline moz_minions/lib/b2g_crash_util.py:1]: suggest we can move this util back to b2g_util ..

**Comment by zapion** [comment]
README updated in #9

File: moz_minions/lib/b2g_crash_util.py (hunk 1 above)
Comment (zapion) [inline moz_minions/lib/b2g_crash_util.py:1]: @askeing ping?^

**Comment by ShakoHo** [comment]
lgtm r+

**Pull request merged**